src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -164,16 +160,15 @@ public function editAction(Post $post, Request $request)
 
         return $this->render('admin/blog/edit.html.twig', [
             'post' => $post,
-            'edit_form' => $editForm->createView(),
-            'delete_form' => $deleteForm->createView(),
+            'form' => $form->createView(),
         ]);
     }
 
     /**
      * Deletes a Post entity.
      *
      * @Route("/{id}", name="admin_post_delete")
-     * @Method("DELETE")
+     * @Method("POST")
```

Comment:
@javiereguiluz enabling it make it easier to create CSRF attacks (well, not for this action as it has a CSRF token). This is why it is disabled by default in Symfony.

and it brings no value (the browser is not doing a DELETE request) and can become a nightmare when dealing with cache proxies (because it is a POST request, and you cannot add a Vary on this override to use the right cache.